packages/payload/package.json

```diff
@@ -74,7 +74,7 @@
     "build": "rimraf .dist && rimraf tsconfig.tsbuildinfo && pnpm copyfiles && pnpm build:types && pnpm build:swc && pnpm build:esbuild",
     "build:esbuild": "echo skipping esbuild",
     "build:swc": "swc ./src -d ./dist --config-file .swcrc --strip-leading-paths",
-    "build:types": "tsc --emitDeclarationOnly --outDir dist",
+    "build:types": "concurrently --group \"tsc --emitDeclarationOnly --outDir dist\" \"tsc-strict\"",
```

Comment:
Typecheck is quite slow. I'm using concurrently for cross-platform parallelization. The --group flag is to print the console output serially even though the commands are executed in parallel